docs/templates/robot.rst

```diff
@@ -0,0 +1,61 @@
+.. _robot:
+
+Using the Robot
+===============
+
+XPD is equipped with a sample loading robot, which requires special syntax to use properly.
+
+0. Load data into ``bt`` as usual
+----------------------------------
+
+Follow the normal routine for loading data from the sample spreadsheet.
+
+1. Tell ``bt`` about where the samples are loaded in the robot magazine
+-----------------------------------------------------------------------
+
+To use the robot properly ``bt`` needs to know where the samples are loaded.
+This is achieved by running ``bt.robot_location_number()``.
+This will prompt you to enter the location for each sample currently in
+``bt``.
+If the sample is not in the magazine that's ok just leave the prompt
+ blank and hit ``<enter>``.
+
+
+2. Run ``xrun`` with the robot flag
+-----------------------------------
+
+To run the scans run
+``run(<list of sample numbers>, <scan number>, robot=True)``
+
+For example
+
+    .. code-block:: python
+
+       xrun([3, 4, 5], 2, robot=True)
+
+
+Advanced usage
+--------------
+
+The robot and ``xrun`` are more flexable than just running one scan.
+Multiple scans can be run per sample.
+
+For example here is a run that runs 3 different plans per sample
+
+    .. code-block:: python
+
+       xrun([3, 3, 4, 4, 5, 5], [2, 5]*3, robot=True)
+
+Note that we multiplied the two scans by three to produce a total of six scans,
+two per sample.
+
+The number of scans doesn't need to be a multiple of the number of samples.
+
+    .. code-block:: python
+
+       xurn([3, 4, 5, 5], [2, 2, 2, 5], robot=True)
```

Comment:
honestly, I would give this as the first example, since it is the most intuitive (i.e., a scan defined per sample).

Then I would define the other usage above.  For the other usage, `[2,5]*3` the logic is not completely clear.  Please add some words that explain what will happen if you type that.  For example, does it run those six scans on each sample, so it would run `(3,2), (3,5), (3,2),(3,5),(3,2),(3,5),(3,2), (3,5), (3,2),(3,5),(3,2),(3,5),(4,2),(4,5)....`
You may want to use a less repetitive example of the behavior too, as the example.